tests/commands/test_bump_recipe.py

```diff
@@ -0,0 +1,59 @@
+"""
+:Description: Tests the `bump-recipe` CLI
+"""
+
+from click.testing import CliRunner
+from pyfakefs.fake_filesystem import FakeFilesystem
+
+from conda_recipe_manager.commands import bump_recipe
+from conda_recipe_manager.commands.utils.types import ExitCode
+from tests.file_loading import get_test_path
+from tests.smoke_testing import assert_cli_usage
+
+
+def test_usage() -> None:
+    """
+    Smoke test that ensures rendering of the help menu
+    """
+    assert_cli_usage(bump_recipe.bump_recipe)
+
+
+def test_bump_recipe_cli(fs: FakeFilesystem) -> None:
```

Comment:
I would parameterize this test so we can validate against many cases. We should make sure that we can increment the build number when it is any number, not just `0`.

Some common test build numbers I would use: `0`, `1`, `-1`, `42`